generators/server/templates/_build.gradle

```diff
@@ -369,7 +373,29 @@ task wrapper(type: Wrapper) {
 
 task stage(dependsOn: 'bootRepackage') {
 }
+<% if(!skipClient) { %>
+if (project.hasProperty('nodeDl')) {
```

Comment:
im not happy with the property name can we have something better may be like `nodeInstall` or `localNode` or something